insights/parsers/tests/test_ip.py

```diff
@@ -88,6 +90,65 @@ def test_ip_addr():
         assert iface is not None
 
 
+IP_S_LINK = """
+1: enp0s3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT qlen 1000
+    link/ether 08:00:27:4a:c5:ef brd ff:ff:ff:ff:ff:ff
+    RX: bytes  packets  errors  dropped overrun mcast
+    1113685    2244     0       0       0       0
+    TX: bytes  packets  errors  dropped carrier collsns
+    550754     1407     0       0       0       0
+2: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN mode DEFAULT
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    RX: bytes  packets  errors  dropped overrun mcast
+    884        98       0       0       0       0
+    TX: bytes  packets  errors  dropped carrier collsns
+    884        10       0       0       0       0
+3: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT qlen 1000
+    link/ether 08:00:27:db:86:9e brd ff:ff:ff:ff:ff:ff
+    RX: bytes  packets  errors  dropped overrun mcast
+    0          1        0       0       0       0
+    TX: bytes  packets  errors  dropped carrier collsns
+    0          4        0       0       0       0
+4: enp0s9: <BROADCAST,UP,MULTICAST> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT qlen 1000
+    link/ether 08:00:27:a6:bd:65 brd ff:ff:ff:ff:ff:ff
+    RX: bytes  packets  errors  dropped overrun mcast
+    0          8        0       0       0       0
+    TX: bytes  packets  errors  dropped carrier collsns
+    0          12        0       0       0       0
+""".strip()
+
+
+class TestIpLinkInfo(unittest.TestCase):
```

Comment:
Can you check with this output too https://gist.github.com/psachin/bef91c3507df426b73bd032246d95ad6

It fails with 
```
$ py.test insights/parsers/tests/test_ip.py 
==================================================================================== test session starts =====================================================================================
platform linux2 -- Python 2.7.14, pytest-3.2.1, py-1.4.34, pluggy-0.4.0
rootdir: /home/psachin/github/insights/insights-core, inifile: setup.cfg
collected 9 items                                                                                                                                                                             

insights/parsers/tests/test_ip.py ..F......
================================================================================== short test summary info ===================================================================================
FAIL insights/parsers/tests/test_ip.py::test_ip_s_link

========================================================================================== FAILURES ==========================================================================================
_______________________________________________________________________________________ test_ip_s_link _______________________________________________________________________________________

    def test_ip_s_link():
        if_list = ip.IpLinkInfo(context_wrap(IP_S_LINK_2)).active
>       assert len(if_list) == 8
E       AssertionError: assert 6 == 8
E        +  where 6 = len(['ppp0', 'lo', 'tun0', 'enp0s25', 'vnet0', 'virbr0'])

insights/parsers/tests/test_ip.py:200: AssertionError
============================================================================= 1 failed, 8 passed in 0.83 seconds =============================================================================
```